src/canopy/knowledge_base/knowledge_base.py

```diff
@@ -109,6 +105,7 @@ def __init__(self,
                  chunker: Optional[Chunker] = None,
                  reranker: Optional[Reranker] = None,
                  default_top_k: int = 5,
+                 pinecone_client: Optional[Pinecone] = None
```

Comment:
This should probably be the 3rd argument, right after `index_name` (just for readability - the order doesn't really matter since we don't allow unnamed args anyway)

Reply:
Moved it 👍